docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -129,16 +121,16 @@ This example shows a flat blob listing, but you can also perform a hierarchical
 
 The sample defines an asynchronous method, using an `async` block.
 
-[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L87-L104)]
+[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L82-L99)]
```

Comment:
Whilst this sample does indeed use an async block, there are no async calls anywhere inside it; this is similar to making a method in C# with the async keyword returning e.g. Task<'T> but there's no use of `await` anywhere inside.

I would change this to either use asynchrony properly, or removing the async completely as currently it probably stands a better chance of confusing people more.